src/main/java/edu/bu/metcs673/trackr/service/impl/BankAccountServiceImpl.java

```diff
@@ -78,4 +78,18 @@ public BankAccount verifyUser(TrackrUser user, long id) {
 		return bankAccount;
 	}
 
+
+
+	//Coded by Xiaobing Hou. YOu can delete or change this part
```

Comment:
You can remove this comment in all classes. Please provide java docs for each method

Reply:
OK, have done